ThemeConfig.css

```diff
@@ -0,0 +1,6 @@
+/* :root {
```

Comment:
What is this file for? It looks like `globals.css`, but I don't see it referenced anywhere.

Reply:
Oops that was supposed to be deleted in another commit!